packages/api/src/services/google.calendar.ts

```diff
@@ -0,0 +1,217 @@
+import { type Event, EventResponseStatus, type Attendee } from "@agreeto/db";
```

Comment:
Is this mainly from agreeto/backend or taken a bit from [cal.com's calendar service](https://github.dev/calcom/cal.com/blob/0cb278d1098dc7405849c7328767fdefbf149213/packages/lib/CalendarService.ts#L65-L66)?

Just want to make sure that if we use cal.com stuff, that we properly reference it to give credit.

Reply:
I think this was an attempt to making the services more FP but I'm not even certain I ended up using them or if i just copy-pasted [this](https://github.com/agreeto/backend/blob/dev/src/event/services/google.calendar.ts) to [here](https://github.com/agreeto/agreeto/blob/prisma-trpc/packages/api/src/services/google.calendar.ts).